src/CONST.ts

```diff
@@ -878,6 +878,7 @@ const CONST = {
         HERE_TEXT: '@here',
     },
     COMPOSER_MAX_HEIGHT: 125,
+    CHAT_FOOTER_SECONDARY_ROW_HEIGHT: 25,
```

Comment:
This value has to be kept in sync with something, right?

Reply:
I didn't have any solution in mind yet. Sometimes, a comment like "This has to be kept in sync with..." can work. I'll see what you figured out in a moment.